Assets/Treasure/TDK/Runtime/Infrastructure/TDKLogger.cs

```diff
@@ -36,6 +34,10 @@ public static void Log(string message)
         }
 
         private static void LogByLevel(TDKConfig.LoggerLevelValue loggerLevelValue, string message) {
+            if (TDK.Instance.appIsQuitting) {
```

Comment:
This is accessing `Instance` which would be destroyed by the time the app is quitting, meaning it will create a new instance and add it to the scene like before, causing the extra game object to stay in the scene. I just tried the code in develop and the originally reported issue still happens.